tests/main/interfaces-microstack-support/test-snapd-sh/bin/inspect

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
+
+cat /proc/self/cgroup > "$SNAP_COMMON/proc_self_cgroup"
+sleep 120
```

Comment:
why the sleep here? shouldn't it go in the retry command arguments?

Reply:
This sleep is to make sure that the program does not terminate before we have inspected the cgroup filesystem: since systemd destroyes the cgroup after the service terminates, if we didn't have this sleep the test code would not be able to reliably check whether the cgroup is properly setup or not.